src/routes/io/io.py

```diff
@@ -80,3 +79,44 @@ async def analyze_img_file(
             "deduced-mime-type": deduced_type})
     except:
         raise internal_error_500()
+
+file_data = {}
+  
+@io_router.post('/user/create')
+async def create_user(username:str
+,    email:str
+,    password:str
+):
+    try:
+        file_data[username] = {
+            "username" : username,
+            "email" : email,
+            "password": password
+        }
+        return ok_200({"username":username,
+                       "email" : email,
+                       "password": password})
+    except:
+        raise internal_error_500()
+
+@io_router.put('/user/update')
+async def update_user(username:str
+,    email:str
+,    password:str
+,    new_email:str
+,    new_password:str):
+    try:
+        old_file_data = file_data[username]
+        new_file_data= {
```

Comment:
new_file_data should be such that:
- if new_email is empty or None then the existing will be used.
- if new_password is empty or None then the existing will be used.